AUMC_Example/pre_MEDS.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+
+"""Performs pre-MEDS data wrangling for AUMCdb.
+
+See the docstring of `main` for more information.
+"""
+
+from collections.abc import Callable
+from datetime import datetime
+from pathlib import Path
+
+import hydra
+import polars as pl
+from loguru import logger
+from omegaconf import DictConfig, OmegaConf
+
+from MEDS_transforms.utils import get_shard_prefix, hydra_loguru_init, write_lazyframe
+
+ADMISSION_ID = "admissionid"
+PATIENT_ID = "patientid"
+
+def load_raw_aumc_file(fp: Path, **kwargs) -> pl.LazyFrame:
+    """Load a raw AUMCdb file into a Polars DataFrame.
```

Comment:
nitpick: For a function this simple, a one-line docstring is reasonable too, imo.

Reply:
Added example logic